pygyver/etl/dw.py

```diff
@@ -776,6 +779,25 @@ def update_schema(self, table_id, schema_path, dataset_id=bq_default_dataset(),p
             except exceptions.BadRequest as error:
                 raise error
 
+
+    def update_table_description(self, table_id, table_description,
+                                 project_id=bq_default_project(),
+                                 dataset_id=bq_default_dataset()
+                                 ):
+        dataset_ref = self.get_dataset_ref(dataset_id, project_id)
+        table_ref = dataset_ref.table(table_id)
```

Comment:
can be simplified to 

```suggestion
        table_ref = self.get_table_ref(dataset_id=dataset_id, table_id=table_id, project_id=project_id)
```

Reply:
i have considered that, but it is not that readable. I think it is better to be more explicit.

Reply:
We need to make the code base consistent. We created the method `get_table_ref` to simplify the code and it is used in most methods needing to get the table ref.